### PR TITLE
Add Jupyter Trainings for both OpenMP Offload

### PR DESCRIPTION
**Pull request by karlqi2000**
Signed-off-by: Karl Qi <karl.qi@intel.com>

# Description

These are Jupyter Notebooks for Training Teach OpenMP Offload for both C++ and Fortran.
They've been on the DevCloud for a while.

## Type of change
New

# How Has This Been Tested?

These have been hosted on the DevCloud for a while and has gone through many User Tests.

- [x] Command Line
- [x] oneapi-cli

# Checklist for Moving samples:
Links and Details can be found in the samples WG Teams Files. 

- [ ] Review sample design with domain reviewers https://github.com/oneapi-src/oneAPI-samples/wiki/Reviewers-and-Domain-Experts 
- [x] Implement coding guidelines and ensure code quality.
- [x] Adhere to sample.json specification. https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-specification
- [x] Run jsonlint on sample.json to verify json syntax. www.jsonlint.com
- [x] Adhere to readme template 
- [ ] Ensure/create CI test configurations for sample (ciTests field) https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-ci-test-object
- [ ] Enforce format via clang-format config file
- [ ] Review DPC++ code with Paul Peterseon. (GitHub User: pmpeter1)
- [ ] Review readme with Tom Lenth or Joe Oster. (GitHub User: JoeOster)
- [x] Tested using Dev Cloud when applicable
- [ ] Implement fixes for ONSAM Jiras
- [ ] If you have new dependencies/binaries, inform Samples Project Manager Swapna R Dontharaju (@srdontha) or @JoeOster

### CODE REVIEW

File: DirectProgramming/C++/Jupyter/OpenMP-offload-training/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+The Jupyter notebooks are tested and can be run on Intel Devcloud.
+Below are the steps to access these Jupyter notebooks on Intel Devcloud
+1. Register to Intel devcloud 
+    https://intelsoftwaresites.secure.force.com/devcloud/oneapi
+2. Go to the "Terminal" in the Intel Devcloud
+3. Type in the below command to download the oneAPI-essentials series and OpenMP offload notebooks into your devcloud account
+    /data/oneapi_workshop/get_jupyter_notebooks.sh
+
+Contact for this repo: Karl Qi karl.qi@intel.com @karlqi2000
```
Comment (JoeOster) [inline DirectProgramming/C++/Jupyter/OpenMP-offload-training/README.md:13]: Please remove personal information

File: DirectProgramming/Fortran/Jupyter/OpenMP-offload-training/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+The Jupyter notebooks are tested and can be run on Intel Devcloud.
+Below are the steps to access these Jupyter notebooks on Intel Devcloud
+1. Register to Intel devcloud 
+    https://intelsoftwaresites.secure.force.com/devcloud/oneapi
+2. Go to the "Terminal" in the Intel Devcloud
+3. Type in the below command to download the oneAPI-essentials series and OpenMP offload notebooks into your devcloud account
+    /data/oneapi_workshop/get_jupyter_notebooks.sh
+
+Contact for this repo: Karl Qi karl.qi@intel.com @karlqi2000
```
Comment (JoeOster) [inline DirectProgramming/Fortran/Jupyter/OpenMP-offload-training/README.md:13]: Please remove personal information

File: DirectProgramming/C++/Jupyter/OpenMP-offload-training/intro/simple_solution.cpp [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        is_cpu=omp_is_initial_device();
+#pragma omp parallel for
+        for (int i=0; i<N; i++) {
+            data[i] *= 2;
+        }
+    }
+
+    //# Print Output
+    printf ("Running on %s\n", (is_cpu?"CPU":"GPU"));
+    for(int i=0; i<N; i++) std::cout << data[i] << std::endl;
```
Comment (JoeOster) [inline DirectProgramming/C++/Jupyter/OpenMP-offload-training/intro/simple_solution.cpp:29]: please use \n vs std::emdl unless there is a specific reason not to

**Review by JoeOster — CHANGES_REQUESTED** [review]
No license file was found for each of the samples, please add the license file from https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt

File: DirectProgramming/C++/Jupyter/OpenMP-offload-training/intro/simple_solution.cpp [hunk 4]
```diff
@@ -0,0 +1,33 @@
+//==============================================================
+// Copyright © 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+#include <omp.h>
+#include <iostream>
+
+static const int N = 16;
```
Comment (pmpeter1) [inline DirectProgramming/C++/Jupyter/OpenMP-offload-training/intro/simple_solution.cpp:9]: should be constexpr

**Review by pmpeter1 — CHANGES_REQUESTED** [review]
main.cpp: It's not best practice to use #define ARRAY_SIZE for constant in C++.    This should be constexpr to define the datatype for this constant.

All code files should be run through clang-format to make sure they are constant (usually -style=google).     saxpy_func_solution_length.cpp looks wrong.

simple_solution.cpp.   Would be better to use C++ style IO and not mix printf and cout.

File: DirectProgramming/C++/Jupyter/OpenMP-offload-training/intro/simple_solution.cpp (hunk 3 above)
Comment (karlqi2000) [reply DirectProgramming/C++/Jupyter/OpenMP-offload-training/intro/simple_solution.cpp]: Done

File: DirectProgramming/C++/Jupyter/OpenMP-offload-training/intro/simple_solution.cpp (hunk 4 above)
Comment (karlqi2000) [reply DirectProgramming/C++/Jupyter/OpenMP-offload-training/intro/simple_solution.cpp]: Done

File: DirectProgramming/Fortran/Jupyter/OpenMP-offload-training/README.md (hunk 2 above)
Comment (karlqi2000) [reply DirectProgramming/Fortran/Jupyter/OpenMP-offload-training/README.md]: Done

**Review by JoeOster — APPROVED** [review]
approved for ci

**Pull request merged**